plugins/calculator/main.go

```diff
@@ -0,0 +1,62 @@
+package calculator
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/aerogear/charmil/pkg/pluginloader"
+
+	"github.com/spf13/cobra"
+)
+
+var AddCommand = pluginloader.CreateCommand(&pluginloader.CommandConfig{
+	Name: "add",
+	Args: []string{"num1", "num2"},

```

Comment:
What approach you use for arguments

Reply:
cobra accepts the arguments in array of strings `args []string`. So I kept it that way. May be we can add NumArgs option for the user.
Any other thoughts?

There might me some commands which sometimes require 1 arg sometimes 2 args. Then specifying exact number of args will be wrong!

Reply:
my take is that wrapper will not work ad it will require significant refactor to integrate existing commands. Some of those will not work with wrapper assumptions which means that we need to break API etc. instead of pushing towards something that is broken we can join forces and do proper cobra based plugin